docs/guides/working-with-multiple-devs.md

```diff
@@ -0,0 +1,63 @@
+# Working with Multiple Developers
+
+When collaborating with multiple developers on a Lorekeeper project, it's essential to establish clear guidelines and best practices to ensure effective collaboration and minimize conflicts.
+
+## The Golden Rules
+
+These are written in absolutes, but they are just guidelines to help you and your hired devs work effectively together. Feel free to use as many or as little as you like, whatever works best for your team!
+
+### **For the Project Manager / Lead:**
+
+- Define clear tasks for each developer. Set deadlines as needed.
+- Use a shared repository separate from your production site (e.g. GitHub).
+- Use a kanban board or similar tool to track progress and tasks. (ex. Trello, GitHub issues, Discord threads, whatever works for your team)
+- If possible, limit access to your live site to only those who absolutely need it.

```

Comment:
Maybe `server/live site`? To be clearer... I'm not sure this is the best way to go about it, but I think it's worth a little additional specificity.